gstreamer-rtsp-server/src/r_t_s_p_token.rs

```diff
@@ -0,0 +1,30 @@
+use RTSPToken;
+use glib::value::ToSendValue;
+use gst;
+use ffi;
+use glib::translate::*;
+
+impl<'a> RTSPToken {
+    pub fn new(values: &[(&str, &ToSendValue)]) -> RTSPToken {
+        assert_initialized_main_thread!();
```

Comment:
This is unneeded, RTSPToken::new_empty() should already have it

Reply:
Done